2.9.0/done/result_mps.md

```diff
@@ -0,0 +1,112 @@
+* Once you are finished, move this very file from `todo/` to `done/` and submit a pull request.
+
+The categories below are as follows:
+
+* BC breaking: All commits that are BC-breaking. These are the most important commits. If any pre-sorted commit is actually BC-breaking, do move it to this section. Each commit should contain a paragraph explaining the rational behind the change as well as an example for how to update user code [BC-Guidelines](https://docs.google.com/document/d/14OmgGBr1w6gl1VO47GGGdwrIaUNr92DFhQbY_NEk8mQ/edit#heading=h.a9htwgvvec1m).  
+* Deprecations: All commits introducing deprecation. Each commit should include a small example explaining what should be done to update user code.  
+* new\_features: All commits introducing a new feature (new functions, new submodule, new supported platform etc)  
+* improvements: All commits providing improvements to existing feature should be here (new backend for a function, new argument, better numerical stability)  
+* bug fixes: All commits that fix bugs and behaviors that do not match the documentation  
+* performance: All commits that are added mainly for performance (we separate this from improvements above to make it easier for users to look for it)  
+* documentation: All commits that add/update documentation  
+* Developers: All commits that are not end-user facing but still impact people that compile from source, develop into pytorch, extend pytorch, etc  
+* not user facing: All commits that are not public end-user facing and hence should be dropped from the release notes
+
+## mps
+
+### bc breaking
+
+- Build metal kernels of MacOS-14+ ([\#159733](https://github.com/pytorch/pytorch/pull/159733))  
+- Remove all pre-MacOS14 logic ([\#159912](https://github.com/pytorch/pytorch/pull/159912))
+
+### deprecation
+
+### new features
+
+### improvements
+
+- Add `shifted_chebyshev_polynomial_[tuvw], simd_[arg][max|min]`,  `igamma/igammac,grid_sampler_3d, native_dropout`/`native_dropout_backward`  ([\#157488](https://github.com/pytorch/pytorch/pull/157488), [\#158990](https://github.com/pytorch/pytorch/pull/158990), [\#161927](https://github.com/pytorch/pytorch/pull/161927), [\#160541](https://github.com/pytorch/pytorch/pull/160541), [\#162108](https://github.com/pytorch/pytorch/pull/162108))  
+- For sparse tensors: Add coalesce, `indices/values,sgn/asinh/atanh/asin/atan/ceil/erf/expm1/floor/frac/isnan/nan_to_num/log1p/rad2deg/deg2rad/neg/round/relu/sin/sinh/sqrt/tan/tanh/sign/signbit/isinf/isposinf/isneginf,cat` ([\#159729](https://github.com/pytorch/pytorch/pull/159729)/[\#160254](https://github.com/pytorch/pytorch/pull/160254), [\#160223](https://github.com/pytorch/pytorch/pull/160223),[\#161846](https://github.com/pytorch/pytorch/pull/161846),[\#162007](https://github.com/pytorch/pytorch/pull/162007))  
```

Comment:
Should we move this one to new feature: and call it something like `[Beta] Partial sparse support for MPS backend`